<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>National Resource for Network Biology</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">

    <!-- Le styles -->
    <link href="assets/css/bootstrap.css" rel="stylesheet">
    <link href="assets/css/bootstrap-responsive.css" rel="stylesheet">
    <link href="assets/css/nrnb.css" rel="stylesheet">

    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Fav and touch icons -->
    <link rel="shortcut icon" href="assets/ico/favicon.ico">
    <link rel="apple-touch-icon-precomposed" sizes="144x144"
          href="assets/ico/apple-touch-icon-144-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="114x114"
          href="assets/ico/apple-touch-icon-114-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="72x72"
          href="assets/ico/apple-touch-icon-72-precomposed.png">
    <link rel="apple-touch-icon-precomposed"
          href="assets/ico/apple-touch-icon-57-precomposed.png">
</head>

<body>


<div class="belt top">
    <div class="container">
        <div class="row top-right">
	  <div class="span6">
        	<h2>Featured Publications
            	<a href="http://netbiopub.tumblr.com/" target="_top">
                <img src="assets/img/tumblr_logo.svg" width="30" /></h2>
        <iframe id="frame" src="netbio_tumblr.html"
                    height="550" width="95%" frameborder="0" scrolling="yes">
            </iframe>	   
		</div>
           <div class="span5">
                <div class="row" float="right">
                    <img class="logo-big" src="images/NRNB_white.svg"/>
                    <h2>National Resource for Network Biology</h2>
                </div>
                <div class="row buttons">
                    <a class="button-flat" float="right" href="projects.html">Research Projects</a>
                    <a class="button-flat" float="right" href="tools.html">Tools</a>
                    <a class="button-flat" float="right" href="training.html">Training</a>
                    <a class="button-flat btn-blue" float="right" href="outreach.html">Collaboration</a>
                    <a class="button-flat btn-green" float="right" href="#renewed">Funding Renewed</a>
                    <a class="button-flat" float="right" href="#highlights">Research Highlights</a>
                </div>
            </div>
        </div>
    </div>
</div>

<div class="belt white">
    <div class="container">
        <div class="row section">
            <h3>What is NRNB?</h3>
        </div>
        <div class="row">
            <div class="span4">
                <iframe width="300" height="225"
                        src="http://www.youtube.com/embed/_nbqSe4Zaps?rel=0
                        &theme=light&color=white"
                        frameborder="0" allowfullscreen>
                </iframe>
            </div>
            <div class="span8">

                <p>
                    The aim of the <b>National Resource for Network Biology (NRNB)</b>
                    is to advance the new science of
                    Biological Networks through analytic tools, visualizations, databases and computing resources.
                    Biomedical research is increasingly dependent on knowledge of biological networks of multiple
                    types and
                    scales, including gene, protein and drug interactions, cell-cell and cell-host communication,
                    and vast
                    social networks. Our technologies enable researchers to assemble and analyze these networks and
                    to
                    use
                    them to better understand biological systems and, in particular, how they fail in disease. NRNB
                    leverages the <a href="http://www.cytoscape.org/">Cytoscape</a> Cyberinfrastructure,
                    a widely-used network analysis
                    platform which we further support through a rich set of tutorials, workshops, 24/7 help desk
                    assistance,
                    and over 100 collaboration and service projects.</p>

                <p> We are developing freely available, open source software
                    technologies to enable network-based visualization, analysis, and
                    biomedical discovery. We are driving research and development <a
                            href="projects.html"><b>projects</b></a>, coordinating <a
                            href="training.html"><b>training</b></a> opportunities, and
                    actively seeking new <a href="outreach.html"><b>collaborations</b></a>
                    to develop NRNB <a href="tools.html"><b>tools</b></a> and apply
                    them to biomedical research.
                </p>
            </div>
        </div>
    </div>
</div>
<div id="renewed"></div>
<div class="belt white">
    <div class="container">
        <div class="row section">
            <h3>5-Year Funding Renewal</h3>
        </div>
        <div class="row">
            <div class="span3">
                <img src="images/NRNB_black.png" alt="nrnb logo">
            </div>
            <div class="span9">

                <p>
                    We have been awarded another five years of funding by <a href="http://publications.nigms.nih.gov/btrrs/
                    searchresultsall.asp">NIGMS</a> for the National Reseource of Network Biology. This was our first 
                    round of competitve renewal as a resource center and thus a major test of both our track record
                    and our vision. We have identified <a href="./projects.html">three major themes</a> in Network Biology 
                    that will define our <a href="./projects.html">technology research and development projects</a>. With 
                    this renewed investment, we will also continue to pursue a broad range of <a href="./outreach">collaboration
                    </a> and <a href="./training.html">training</a> opportunities. 
                </p>
            </div>
        </div>
    </div>
</div>
</div>
<div id="highlights"></div>
<div class="belt white">
    <div class="container">
        <div class="row section">
            <h3>Research Highlights</h3>
        </div>

        <div class="row belt">
            <div class="span3">
                <img src="images/NeXO.png" alt="image">
            </div>
            <div class="span9">
                <p><b>Network Approach to Building Gene Ontologies</b><br/>
                    <a href="http://nexontology.org/">NeXO</a> (The Network Extracted Ontology) uses a principled
                    computational approach which integrates evidence from hundreds of thousands of individual gene and
                    protein interactions to construct a complete hierarchy of cellular components and processes. This
                    data-derived ontology aligns with known biological machinery in the GO Database and also uncovers
                    many new structures.
                </p>
            </div>
        </div>

        <div class="row belt">
            <div class="span3">
                <img src="images/nbs.png" alt="image">
            </div>
            <div class="span9">
                <p><b>Network-based stratification of tumor mutations</b><br/>
                    <a href="http://www.nature.com/nmeth/journal/v10/n11/full/nmeth.2651.html">NBS</a> is a method for stratification (clustering) of patients in a cancer cohort based on somatic mutations and a gene interaction network. The method uses network propagation to integrate genome wide somatic mutation profiles for each patient over a gene interaction network, and a non-negative matrix factorization based clustering approach in order to derive biologically meaningful stratification of a patient cohort. 
                </p>
            </div>
        </div>

        <div class="row belt">
            <div class="span3">
                <img src="images/cyappstore1.png" alt="image">
            </div>
            <div class="span9">
                <p><b>Cytoscape App Store</b><br/>
                    The overarching goals of the <a href="http://apps.cytoscape.org">Cytoscape App Store</a> are to
                    highlight the important features apps add to Cytoscape, to enable researchers to find apps they
                    need, and for developers to promote their apps. The Cytoscape App Store highlights the important
                    features that apps add to Cytoscape, enable researchers to find and install apps they need, and help
                    developers promote their apps.
                </p>
            </div>
        </div>

        <div class="row belt">
            <div class="span3">
                <img src="images/cBioPortal_2.png" alt="image">
            </div>
            <div class="span9">
                <p><b>Network Analysis Tools</b><br/>
                    Our research programs are exploring an array of new approaches to network analysis, such as how to
                    use
                    molecular networks to stratify cancer patient genomes (Network-Based Stratification) or how to
                    assemble
                    a gene ontology of the cell directly from molecular network data (<a
                            href="http://www.nexontology.org/">nexontology.org</a>). In recent years
                    we
                    have released a number of widely-used network analysis resources, including the Cancer Biology
                    Portal
                    (<a href="http://www.cbioportal.org/public-portal/">cBioPortal.org</a>), the GeneMANIA network
                    database (<a href="http://genemania.org/">genemania.org</a>), and the Cytoscape Appstore
                    (<a href="http://apps.cytoscape.org/">apps.cytoscape.org</a>).
                </p>
            </div>
        </div>

        <div class="row belt">
            <div class="span3">
                <img src="images/nrnbPNASfriendNetworks.png" alt="image">
            </div>
            <div class="span9">
                <p><b>Correlated Genotypes in Friendship Networks </b><br/>
                    In his book CONNECTED, NRNB investigator James Fowler argued that "social networks are in our
                    nature." Then last year he published a paper showing that genes influence our social network
                    position -- how central we are, and how likely it is that our friends know one another. In a related
                    NRNB study published in PNAS this year, Fowler examined another important social network process
                    called "homophily" -- it's a word that literally means "love of like" and it refers to the idea that
                    we tend to make friends with people who resemble us -- "birds of a feather flock together."
                </p>
            </div>
        </div>

        <div class="row belt">
            <div class="span3">
                <img src="images/SynthGeneAnalysisYeast.png" alt="image">
            </div>
            <div class="span9">
                <p><b>Synthetic Genetic Analysis of Budding Yeast</b><br/>
                    NRNB investigator Gary Bader has been collaborating with scientists at the University of Toronto
                    (Charles Boone, Brenda Andrews) to complete the first complete genetic interaction network for a
                    cell and to decipher the general principles that govern this network. This reference map provides a
                    model for expanding genetic network analysis to higher organisms, and it will stimulate valuable
                    insights into gene function, drug target and mode-of-action analysis. The resulting complete map of
                    genetic interactions for budding yeast, with ~6000 genes, will contain 36 million quantitative
                    interaction pairs (18 million unique pairs). The most recent publication of roughly 20% of the
                    complete map was in the top 30 most cited papers of 2.2 million in 2010. The map currently is 75%
                    complete and continues to be analyzed.
                </p>
            </div>
        </div>

        <div class="row belt">
            <div class="span3">
                <img src="images/GeneNetworksDNAdamage.png" alt="image">
            </div>
            <div class="span9">
                <p><b>Genetic Networks Remodeled by DNA Damage</b><br/>
                    Although cellular behaviors are dynamic, the networks that govern these behaviors have been mapped
                    primarily as static snapshots. To explore network dynamics, the Ideker laboratory has been
                    collaborating with the laboratory of Nevan Krogan at UCSF to analyze interaction networks as they
                    are remodeled by different cellular stresses and stimuli. This past year, they developed a new
                    approach called differential epistasis mapping (dE-MAP) which creates a genetic network based on the
                    changes in interaction strength observed between two static conditions. Using this approach, they
                    have mapped widespread changes in genetic interaction among yeast kinases, phosphatases, and
                    transcription factors as the cell responds to DNA damage. This proof-of-principle work suggests that
                    differential networks chart a new type of genetic landscape that will be invaluable for mapping many
                    different cellular responses to stimuli.
                </p>
            </div>
        </div>

        <a class="btn button-flat btn-gray pull-right"
           href="http://www.citeulike.org/user/nrnb"
	   target="_blank">
            Recent NRNB Publications
        </a>
    </div>
</div>

<footer></footer>

<!-- For Navbar -->
<script src="assets/js/jquery-1.11.0.min.js"></script>
<script src="assets/js/bootstrap.min.js"></script>
<script src="assets/js/page_setup.js"></script>

<!-- For Google Analytics: DO NOT MODIFY! -->
<script type="text/javascript">
    var _gaq = _gaq || [];
    _gaq.push([ '_setAccount', 'UA-19386719-1' ]);
    _gaq.push([ '_trackPageview' ]);

    (function () {
        var ga = document.createElement('script');
        ga.type = 'text/javascript';
        ga.async = true;
        ga.src = ('https:' == document.location.protocol ? 'https://ssl'
                : 'http://www')
                + '.google-analytics.com/ga.js';
        var s = document.getElementsByTagName('script')[0];
        s.parentNode.insertBefore(ga, s);
    })();
</script>

</body>
</html>
