<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>NRNB: GSoc Ideas Page</title>

<!-- Le styles -->
<link href="../assets/css/bootstrap.css" rel="stylesheet">
<link href="../assets/css/bootstrap-responsive.css" rel="stylesheet">
<link type="text/css" rel="stylesheet" href="../assets/css/jquery.tocify.css" />
<link type="text/css" rel="stylesheet" href="../assets/css/jquery-ui.min.css" />
<style type="text/css">
#nav {
     position: fixed;
}
.tocify ul, .tocify li {
        line-height: 15px;
}
</style>
</head>
<body>
<div class="row" height="100%">
    <div class="span4"  height="100%">
         <!-- Our table of contents will be here !-->
         <div id="nav" style="font-size: 12px;" height="100%"></div>
    </div>
    <div id="ideas" class="span8">

<!-- PASTE HERE -->

<p><br />
</p>

<h2 dir="ltr" >Idea 1: Submit your own original proposal</h2>

<p>Feel free to propose your own idea. As long as it relates to one of our projects, we will give it serious consideration. Creativity and self-motivation are great traits for open source programmers, but make sure your proposal is also relevant.</p>

<br />
<h2 dir="ltr" >Idea 2: Co-authorship Social network App</h2>

<p>Description: Biological networks can be visualized and analyzed using Cytoscape. Often researchers want to go beyond the network of proteins or genes and also look at the inter-connectedness between colleagues and institutions. Who tends to publish together? What institutions are most collaborative? Are there inter-disciplinary connections in my institution?</p>

<br />
<p>Goal: Create a visual summary of how individuals are connected. An existing App currently queries PubMed directly or takes in a file with a set of publications (from scopus or incites) and builds a co-publication network where nodes are authors and edges the number of publications two authors share. Different sources have different depths of data, for instance PubMed, the most widely accessible of the three main sources, has citation counts but they are limited to only those articles cited in pubmed central and its institution data is limited to the first author. Given that PubMed is the most accessible source we want to expand its functionality to include computed institution data as well as better citation count approximations, and H-index calculation for individuals or subsets of papers. </p>

<p>This functionality can be applied to several data types outside academia, whenever relationships between people is of interest. An example is professional social networks (e.g. linked-in), where individuals are connected by who they have worked with in the past or the twitter-verse where individuals are connected when they each reference the same (#) hash tag. This project will start with support for co-publication networks and extend to other networks like twitter and linked-in as time permits. Additionally, we want to expand our co-authorship cytoscape App to include generic file formats to easily create the above networks.</p>

<br />
<p>Links:</p>

<p>Existing code - <a href="https://github.com/BaderLab/SocialNetworkApp" >https://github.com/BaderLab/SocialNetworkApp</a></p>

<p>Existing userguide - <a href="http://baderlab.org/UserguideSocialNetworkApp" >http://baderlab.org/UserguideSocialNetworkApp</a></p>

<br />
<p>Language and Skills: Java, XML, PubMed e-utils</p>

<p>Potential Mentors: Sergio Baranzini, Gary Bader, Ruth Isserlin</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 3: User Interaction on Biomedical Linked Data</h2>

<p>During the GSoC 2012, we developed<a href="http://vsdlc3.blogspot.com" > SemScape</a>, a plugin that allows the use of Cytoscape as a browser for Semantic Web (Linked Data) knowledge bases. We are currently in the process of updating libs and packaging the plugin for the Cytoscape App Store.</p>

<p>The idea for this year’s project is to go one step further and to allow the use of Cytoscape to visually define queries in SPARQL. The basic idea is that from a graph, a sub-graph can be defined where a few nodes or edges can be marked as variables (and eventually filters can be associated to them): this is equivalent to a (simple) SPARQL query. The user can then define queries in a simple and interactive way, without knowing the underlying syntax, that can be stored and applied on different data sources.</p>

<p>The project can then be extended in three directions, depending on the interest of the student:</p>

<ul>
<li><p>allowing users to push such queries to a repositories and hence share them (the current plugin support updates of queries from remote repositories)</p>

</li>
<li><p>Implementing more complex queries (unions, negation, …)</p>

</li>
<li><p>Implementing heuristics for query-selection: in the current plugin, queries can be performed on demand, but they are also associated to specific actions. For instance, right-selecting a node of type protein can present as an option to expend the graph with the results of a query which is can include the specific protein as a value. This is possible through a simple markup through which queries are annotated by the user. Such markup could be included in the visual query/construction process and/or extracted from simple query patterns.</p>

</li>
</ul>

<br />
<p>Some of the ideas in this project<a href="http://www.biomedcentral.com/1471-2105/9/S4/S6" > have been already prototyped</a>, so we have a good understanding of the difficulty and timing required. </p>

<p>This project has the potential to make Cytoscape an ideal tool to navigate and query Linked Data resources (both in biomedical and, more generally, in Open Data domains). It also opens interesting possibilities in analyzing graphs from graph knowledge bases that can be represented as RDF (e.g.: dbpedia, freebase,...).</p>

<br />
<p>Language and Skills: Java/Maven. Knowledge of OSGi would be helpful. Knowledge of Semantic Web technologies would be a plus.</p>

<p>Potential Mentors: Andrea Splendiani, Andra Waagmeester</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 4: Cytoscape plugin for importing and visualizing variation data</h2>

<p>Background: Variation data identifies genomic regions where individual base pairs are different from a reference genome. The consequences of these variations (e.g., stop gained in a protein sequence due to a SNP in a coding region) can influence gene regulation, structure and function, and are thus associated many genetic diseases. These data sets are large and difficult to navigate.</p>

<br />
<p>Goal: To extend and publish <a href="http://www.dishevelled.org/variation-cytoscape3-app" >a Cytoscape 3 app in development</a> that reads in variation data from various services and dynamically maps the data to biological networks and pathways independently (or selectively) loaded into Cytoscape. The app should provide the services needed to perform the mapping as well as default visualization settings, such as coloring all genes with variations in coding regions red.</p>

<br />
<p>The app currently supports:</p>

<ul>
<li><p>Loading variation data from <a href="http://beta.rest.ensembl.org/" >Ensembl REST services</a>, <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41" >VCF</a> files, and <a href="https://github.com/arq5x/gemini" >GEMINI</a></p>

</li>
<li><p>Loading variation consequence data from Ensembl REST services, <a href="http://snpeff.sourceforge.net/" >SnpEff</a>-annotated VCF files, <a href="http://www.ensembl.org/info/docs/tools/vep/" >Ensembl VEP</a>-annotated VCF files, and GEMINI</p>

</li>
<li><p>Predicting variation consequences with Ensembl VEP via Ensembl REST services</p>

</li>
<li><p>Summarizing variations and variation consequences per row/gene by <a href="http://www.geneontology.org/" >Gene Ontology</a> term</p>

</li>
<li><p>Visual mappings for variation and variation consequence counts</p>

</li>
</ul>

<br />
<p>The app should be extended to provide:</p>

<ul>
<li><p>Default color mappings for variation consequences by Sequence Ontology term</p>

</li>
<li><p>Advanced/compound visual mappings for counts of variation consequences summarized at the row/gene level (e.g., mutation diagrams, data rings, donut charts)</p>

</li>
<li><p>Variation service implementation based on <a href="https://github.com/bigdatagenomics/adam" >ADAM</a></p>

</li>
<li><p>Variation service implementation based on a REST client for GEMINI</p>

</li>
<li><p>In-memory variation consequence prediction services based on SnpEff and <a href="http://compbio.charite.de/contao/index.php/jannovar.html" >JAnnovar</a></p>

</li>
<li><p><a href="http://www.hgvs.org/mutnomen/" >HGVS</a> identifiers for variants where available (possibly via <a href="https://github.com/invitae/hgvs" >hgvs mapper</a>)</p>

</li>
<li><p>Compare/contrast variation and/or variation consequences between samples (e.g. tumor vs normal) or populations (e.g. a case/control study)</p>

</li>
</ul>

<br />
<p>Language and Skills: Java/Swing</p>

<p>Potential Mentors: Alex Pico, Paul Tang, Nathan Salomonis, Michael Heuer</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 5: CentiScaPe for multiple networks analysis and graphical output enhancement</h2>

<p>Background: CentiScaPe is a Cytoscape plugin that allows to compute network centrality parameters. It is the most complete Cytoscape plugin for network centralities calculating Degree, Average Shortest Path, Eccentricity, Closeness, Betweenness, Centroid, Stress, Radiality, Eigenvector, Edge Betweenness, Bridging Centrality, for directed and undirected networks. CentiScaPe allows identifying network nodes that are relevant from both experimental and topological viewpoints. CentiScaPe also provides a Boolean logic- based tool that allows easy characterization of nodes whose topological relevance depends on more than one centrality. Finally, different graphic outputs and the included description of biological significance for each computed centrality facilitate the analysis by the end users not expert in graph theory, thus allowing easy node categorization and experimental prioritization.</p>

<br />
<p>Goal: The new beta version of CentiScaPe provides also the possibilities to analyze multiple networks and to integrate the analysis with experimental data set. This implies that a user can automatically start a computation of several networks and waits for the results. Anyway, the output of the analysis of a lot of networks (say 100) is useless if not supported by a smart and user friendly system to analyze the results. The Idea is to implements some techniques used in software analysis to extract information from the results of multiple computation in order to compare different networks and to extract topological based biomarkers:</p>

<ul>
<li><p>The main framework for the results analysis is the Abstract Interpretation theory, particularly the interval analysis (and eventually others) [1].</p>

</li>
<li><p>Some statistical parameters, automatic analysis of the output, and new kinds of graphical outputs that can help to extract information from a multiple networks centrality analysis should be provided. The tools should for example be able to answer to questions as “How does the centroid value of the protein A chenge in the submitted 100 different networks?” or “How does the betweenness of protein B change if we consider 30 different phosporilation values for each proteins in the network N (this means we have 30 different experimental data set for the same network that must be included in the computation)?”.</p>

</li>
<li><p>The tools should also provide the implementation of the integration of experimental data-sets for the eigenvector, bridging centrality and edge betweenness (they are now available for all the other centralities).</p>

</li>
</ul>

<br />
<p>[1] Patrick Cousot and Radhia Cousot. 1977. Abstract interpretation: a unified lattice model for static analysis of programs by construction or approximation of fixpoints. In Proceedings of the 4th ACM SIGACT-SIGPLAN symposium on Principles of programming languages (POPL ‘77). <a href="http://doi.acm.org/10.1145/512950.512973" >http://doi.acm.org/10.1145/512950.512973</a></p>

<br />
<a href="http://doi.acm.org/10.1145/512950.512973" ></a><p>Language and Skills: Java, software analysis (possibly abstract interpretation theory)</p>

<p>Potential Mentors: Giovanni Scardoni (CBMC, University of Verona), Fausto Spoto (University of Verona)</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 6: Develop Heat-map Based Data Views for Cytoscape 3 using CoolMap</h2>

<p>Background: CoolMap is a novel omics data exploration application/library that offers new ways to explore complex molecular data. It is based on the classic heatmap paradigm, but allows users to use generated (such as clustering) or pre-defined (such as gene ontology or sample groups) ontologies to aggregate big datasets into concept-level, human manageable views. The rows and columns of CoolMap can be expanded/collapsed interactively and each cell can be plotted in many ways (color gradient, statistical plots, etc.). It efficiently facilitates the exploratory data analysis, and help researchers generate hypotheses from data.</p>

<p>Paired with other software like Cytoscape, this visualization duo has a lot of potential to transform the knowledge discovery processes from biomedical networks. CoolMap can be a good upgrade candidate for the currently used Java TreeView for Cytoscape.</p>

<br />
<p>A basic Cytoscape to CoolMap plugin framework will be provided to the student. The student then can harness the power of Cytoscape and CoolMap to explore new visualization possibilities.</p>

<br />
<p>CoolMap will public in several weeks and news and updates can be found at <a href="http://www.coolmap.org" >http://www.coolmap.org</a></p>

<br />
<p>Goal: Develop either a Cytoscape plugin using CoolMap’s core visualization library, or a CoolMap plugin that utilizes the full functionality of the CoolMap application to enhance Cytoscape’s heatmap-based visualization capabilities.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Gang Su (University of Michigan, Ann Arbor)</p>

<p>Contact: <a href="mailto:sugang@umich.edu" >sugang@umich.edu</a>, <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 7: Developing An Interface For The Cancer Network Altering Variant Database (CAN-VD) Using Cytoscape.js </h2>

<p>Background: Cancer is highly associated with genomic mutations. Such mutations in genes affect their protein products resulting in mutated proteins that acquire new function (gain-of-interaction/function), mutants that lose their function (loss-of- interaction/function) or mutants that lose a function and acquire another (switch-of-interaction/function). Identifying mutants that are related to cancer development and interpreting their functional consequences is important for understanding cancer biology and developing diagnostics, prognostics and drug targets.</p>

<br />
<p>Focus: We are studying the effects of mutations on protein-protein interaction (PPI) networks in cancer. The effects of over 800,000 missense mutations are being analyzed and stored in the Cancer Network Altering-Variant Database (CNA-VD), a relational database implemented in MySQL. CAN-VD stores the PPI interactions mediated by wildtype and variant protein sequences to build and compare the PPI network in the two conditions and understand the effects of mutations on the network topology and, consequently, the cellular and biological functions of the cancer system.</p>

<br />
<p>Goals:</p>

<ol ><li><p>Develop a web-based interface for PPI data retrieval with a comprehensive search function.</p>

</li>
<li><p>Develop a visualization and annotation module using Cytoscape.js to visualize the retrieved PPI data as a network with annotations that highlight the mutation effects (the differences between the normal and cancer networks). </p>

</li>
</ol><br />
<p>Inputs: The student will be provided with the database design and sample data as well as an introduction to the topic and the different possible visualizations and annotations needed. The student will have the freedom to choose among them or to propose other options.</p>

<br />
<p>Language and Skills: Javascript, JSON, PHP</p>

<p>Potential Mentors: Mohamed Helmy, Gary Bader</p>

<p>Contact: <a href="mailto:mohamed.attiashahata@utoronto.ca" >mohamed.attiashahata@utoronto.ca</a>, <a href="mailto:gary.bader@utoronto.ca" >gary.bader@utoronto.ca</a></p>

<br />
<h2 dir="ltr" >Idea 8: Cytoscape.js Based Viewer for SBGN Process Description Diagrams with Built-In Complexity Management Operations</h2>

<p>Background: </p>

<p>Cytoscape.js is a JavaScript graph library for analysis and visualization of relational information [1]. The Systems Biology Graphical Notation (SBGN) is a visual language developed by the community with the intention of enabling scientists to represent networks, including models of cellular processes, in a standard, unambiguous way [2]. SBGN comprises three complementary languages suited to represent different types of pathways. SBGN Process Descriptions is a language that permits the description of the processes (e.g. biochemical reactions) taking place in a biological system and makes heavy use of compound structures (e.g. compartments and molecular complexes). Support for compound structures has been added to Cytoscape.js core recently.</p>

<p>Often times, SBGN PD networks are too large to visualize and details become obscured. Strategies to conceal the "unwanted" details of a network, allowing one to temporarily remove certain nodes and edges and subsequently reintroduce the removed details. Complexity management techniques in graph visualization such as filtering, hiding, folding, and nesting play an important role in effective analysis of biological networks as much as any other domain.</p>

<br />
<p>Goal:</p>

<p>Currently we are in the process of customizing Cytoscape.js to display biological networks drawn in SBGN Process Description notation. The goal of this project is to help with any work left for this purpose and then implement necessary operations to properly manage the complexity of SBGN PD networks.</p>

<br />
<p>[1] <a href="http://cytoscape.github.io/cytoscape.js/" >http://cytoscape.github.io/cytoscape.js/</a></p>

<p>[2] <a href="http://www.sbgn.org/" >http://www.sbgn.org/</a></p>

<br />
<p>Language and Skills: Java, JavaScript</p>

<p>Potential Mentors: Ugur Dogrusoz (Bilkent University) and S. Onur Sumer (MSKCC)</p>

<p>Contact: <a href="mailto:ugur@cs.bilkent.edu.tr" >ugur@cs.bilkent.edu.tr</a> and <a href="mailto:onur@cbio.mskcc.org" >onur@cbio.mskcc.org</a> </p>

<br />
<h2 dir="ltr" >Idea 9: Advanced data visualization in PathVisio</h2>

<p><a href="http://pathvisio.org/" >PathVisio</a> is a pathway editor and analysis tool. PathVisio allows users to upload and visualize data on biological pathways. We want to extend the visualization module in PathVisio which is part of the PathVisio core application. A central visualization registry should allow plugins to create and register new visualization options. The current visualization interface in PathVisio is very complex and should be made more user-friendly and intuitive for the users. </p>

<br />
<p>Goals:</p>

<p>- Independent visualization module and registry</p>

<p>- More user-friendly visualization interface</p>

<p>- Possibility for users to define visualization templates that can be used for different datasets (currently a visualization is linked to a dataset and has to be created again for a new dataset)</p>

<br />
<p>Language and Skills: Java, OSGi</p>

<p>Potential Mentors: Martina Kutmon, Anwesha Bohler (Unimaas, NL)<br />
Contact: <a href="mailto:wikipathways-discuss@googlegroups.com" >wikipathways-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 10: Advanced data import in PathVisio</h2>

<p><a href="http://pathvisio.org/" >PathVisio</a> is a pathway editor and analysis tool. PathVisio allows users to upload and visualize data on biological pathways. Currently it is possible to upload only one expression dataset at a time as a comma-delimited file. This functionality should be extended to allow uploading a number of data files at the same time as well as uploading of multiple data file formats (for eg. Excel sheets).</p>

<p>This will involve extending the <a href="http://developers.pathvisio.org/browser/trunk/modules/org.pathvisio.gex" >gex</a> and <a href="http://developers.pathvisio.org/browser/trunk/modules/org.pathvisio.data" >data</a> modules of PathVisio. These modules are part of the PathVisio core and take care of importing experimental data and mapping it to the elements in a pathway. </p>

<p>The modules need to be extended so as to allow:</p>

<ul>
<li><p>Uploading of multiple expression datasets at once.</p>

</li>
<li><p>Linking each individual dataset to a particular element type, for eg. linking a gene expression dataset to genes in a pathway and a flux data set to the interactions in a pathway.</p>

</li>
<li><p>Importing data from multiple file formats (For eg: .xls, .ods).</p>

</li>
<li><p>The “Data” tab in the side panel of PathVisio will show all the currently loaded datasets, the biological elements they are linked to, and the currently loaded annotation databases.</p>

</li>
</ul>

<br />
<p>Language and Skills: Java, OSGi</p>

<p>Potential Mentors: Anwesha Bohler, Martina Kutmon (Unimaas, NL)</p>

<p>Contact: <a href="mailto:wikipathways-discuss@googlegroups.com" >wikipathways-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 11: Enhance hierarchical models within Segway</h2>

<p><a href="http://noble.gs.washington.edu/proj/segway/" >Segway</a> performs semi-automated annotation of the genome based on functional genomics data (ChIP-seq, RNA-seq, DNase-seq). Essentially it performs unsupervised clustering of these data anchored to the genome, using a dynamic Bayesian network framework. Since different patterns are visible at different scales, we could best describe genomic behavior using a hierarchical model, which would also allow us to better control additional parameters controlling transitions between segment labels and the emission of data. This would enable us to change the representation the relationship between these patterns from a simple network to a network of networks. Initial implementation of two-level hierarchical segmentation is included in Segway, but needs some improvement, such as:</p>

<ul>
<li><p>add ability to do Viterbi decoding of both sublabels and superlabels separately</p>

</li>
<li><p>change superlabels so they emit sublabels instead of strict 1-to-many superlabel-sublabel transition</p>

</li>
<li><p>enable the use of hierarchical models in semi-supervised learning mode</p>

</li>
<li><p>add ability to enforce left-to-right movement through sequence</p>

</li>
</ul>

<br />
<p>Language and Skills: Python, Dynamic Bayesian Networks</p>

<p>Potential Mentors: Michael Hoffman</p>

<p>Contact: <a href="mailto:mhoffman@uhnresearch.ca" >mhoffman@uhnresearch.ca</a></p>

<br />
<h2 dir="ltr" >Idea 12: New Features for Interference App</h2>

<p>Background: Interference is a Cytoscape plug-in for virtual knock-out experiments on complex networks. It allows to evaluate the topological effects of single or multiple nodes removal from a network. In this perspective, interference allows virtual node knock-out experiments: it is possible to remove one or more nodes from a network and analyze the consequences on network structure, by looking to the variations of the node centralities values. As the centrality value of a node is strictly dependent on the network structure and on the properties of other nodes in the network, the consequences of a node deletion are well captured by the variation on the centrality values of all the other nodes. With the interference app the user can:</p>

<ul>
<li><p>Simulate pharmacological treatment: one can potentially predict side effects of the drug by looking at the topological properties of nodes in a drug-treated network, meaning with that a network in which a drug-targeted node (protein) was removed. To inhibit a protein (for instance a kinases) corresponds to removing the node from the network</p>

</li>
<li><p>Simulate gene deletion: gene deletion implies losing encoded proteins, thus resulting in the corresponding removal of one or more nodes from a protein network</p>

</li>
</ul>

<br />
<p>Goal: The Goal of the proposal is to add more features to the App in order to consistently improving the tool:</p>

<ol ><li><p>Calculation of centralities and other new node descriptors (node degrees, Markov Shannon entropies). These numbers will be used to rank the nodes for elimination; the user should choose the criteria (what node descriptor to be use for ranking, the default should be the node degree).</p>

</li>
<li><p>Ranking the nodes using one node descriptor</p>

</li>
<li><p>Eliminate 1 - x nodes and calculate each elimination step your entire network descriptors (ex: average node descriptors) and the Markov Shannon entropy (for the entire node) ; the user could choose the % or number of nodes to keep at the end of the network attack (ex: he can eliminate only 30% of the nodes using the node degree ranking)</p>

</li>
<li><p>Presenting the results as text/table and plot.</p>

</li>
<li><p>Interactive plot: the user can select one eliminated node into the final plot and see metabolic info, notations, semantics, etc. (the normal information you can have in Cytoscape)</p>

</li>
</ol><br />
<p>More info at:<a href="http://www.cbmc.it/%7Escardonig/interference/Interference.php" > http://www.cbmc.it/%7Escardonig/interference/Interference.php</a></p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Cristian Robert Munteanu (University of A Coruña)</p>

<p>Potential Co-Mentors: Giovanni Scardoni (CBMC, University of Verona), Fausto Spoto (University of Verona)</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a>, <a href="mailto:muntisa@gmail.com" >muntisa@gmail.com</a></p>

<br />
<h2 dir="ltr" >Idea 13: Develop a Method for Aligning Pathways and Visualizing these Alignments</h2>

<p>There is continued growth in the groups generating machine-readable representations of biological pathways (such as <a href="http://www.reactome.org/" >Reactome</a> or <a href="http://wikipathways.org/" >WikiPathways</a>) and subsequent efforts to aggregate these pathways into unified repositories, such as Pathway Commons. Aggregation of these pathways suffers from multiple pathways being aggregated that differ due to mismatches in author representations of these pathways. The goal of this project is to develop a method for aligning pathways and then visualizing the results of these alignment. Starting points could be to find interactions that use the same substrates and produce the same products. </p>

<br />
<p>There are many uses of such a tool including helping to validate pathways as they are added to repositories, merging pathways across different resources, and the comparative analyses of pathways across organisms. One way to think about this project, is if you're given multiple representations of the same pathway (<a href="http://wikipathways.org//index.php?query=wnt&amp;species=Homo+sapiens&amp;title=Special%3ASearchPathways&amp;doSearch=1&amp;ids=&amp;codes=&amp;type=query" >examples of several human Wnt pathways</a>) can you develop a method to show a unified pathway and how the original representations map to the unified pathway. </p>

<br />
<p>Language and Skills: Java, XML</p>

<p>Potential Mentors: Augustin Luna (MSKCC)</p>

<p>Contact: <a href="mailto:lunaa@cbio.mskcc.org" >lunaa@cbio.mskcc.org</a></p>

<br />
<h2 dir="ltr" >Idea 14: Add Additional Graph Analysis Algorithms to Cytoscape.js</h2>

<p><a href="http://cytoscape.github.io/cytoscape.js/" >Cytoscape.js</a> is a Javascript graph library that can be used for both the analysis and visualization of biological networks. Currently, Cytoscape.js is missing additional functions for the analysis of graphs. For instance, Cytoscape.js only has a Breadth First Search algorithm implemented for the traversal of graphs. Additional, methods could also be implemented to include additional functions to calculate statistics about graphs. The goal is to augment the graph analysis features in Cytoscape.js implemented purely in Javascript.</p>

<br />
<p>Language and Skills: Javascript, Graph Theory</p>

<p>Potential Mentors: Augustin Luna (MSKCC)</p>

<p>Contact: <a href="mailto:lunaa@cbio.mskcc.org" >lunaa@cbio.mskcc.org</a></p>

<br />
<h2 dir="ltr" >Idea 15: Work on the Pathway Database Converters for the Expansion of Pathway Commons</h2>

<p><a href="http://www.pathwaycommons.org/about/" >Pathway Commons</a> (PC) aggregates biological pathway information from several pathway databases; the data is stored primarily in the format known as <a href="http://www.biopax.org/" >BioPAX</a>. We are continually seeking to expand the amount of data present in Pathway Commons, while maintaining the high quality of data in PC. Databases that have been proposed for future inclusion are the <a href="http://ctdbase.org" >Comparative Toxicogenomics Database</a> (CTD) and <a href="http://humanmetabolism.org/" >RECON</a>, a human metabolic database. </p>

<br />
<p>Each of these databases, has its own challenges for conversion to BioPAX and future inclusion into PC. Inclusion of CTD would require the student to develop a mapping of CTD to BioPAX concepts, and develop a converter using <a href="http://www.biopax.org/paxtools.php" >Paxtools</a>, a BioPAX API written in Java. RECON is currently in the <a href="http://sbml.org/Main_Page" >SBML</a> format and would require conversion to BioPAX. The goal of this project is to develop a converter for one of these databases (or another that the student proposes) and convert the database; the output of the converter should validate using the <a href="http://webservice.baderlab.org:28080/home.html;jsessionid=070EBE4AF7FC1EF04E7D8599F2488A1B" >BioPAX Validator</a>.</p>

<br />
<p>Language and Skills: Java, XML</p>

<p>Potential Mentors: Augustin Luna (MSKCC)</p>

<p>Contact: <a href="mailto:lunaa@cbio.mskcc.org" >lunaa@cbio.mskcc.org</a></p>

<br />
<h2 dir="ltr" >Idea 16: Pathvisiojs (JavaScript) Diff Viewer for Biological Pathway Diagrams</h2>

<p><a href="https://github.com/wikipathways/pathvisiojs" >Pathvisiojs</a> is a web-based viewer and editor for biological pathway diagrams. It was developed by the <a href="http://wikipathways.org/" >WikiPathways.org</a> team to enable students and researchers to browse, draw, analyze, download, publish and share pathways. A specific need for curation is the ability to compare two different pathways or two different versions of the same pathway. This project calls for performing an efficient diff on the JSON data of two pathways and rendering the result as an easy to understand SVG image, with visual cues highlighting the differences. The resulting image should be interactive to allow the user to click on highlighted differences to view detailed information.</p>

<br />
<p>A Diff Viewer already exists, but needs to be replaced as its based on obsolete Java Applet technology. Follow the link to view an example of a comparison of two versions of the same pathway. Note that all highlighted objects are clickable.</p>

<p><a href="http://www.wikipathways.org//index.php?title=Special%3ADiffAppletPage&amp;pwTitle=Pathway%3AWP4&amp;new=73615&amp;old=72246" >http://www.wikipathways.org//index.php?title=Special%3ADiffAppletPage&amp;pwTitle=Pathway%3AWP4&amp;new=73615&amp;old=72246</a> (recommend Firefox with applet support extension)</p>

<br />
<p>Language and Skills: JavaScript, D3.js, SVG</p>

<p>Potential Mentors: Anders Riutta, Alex Pico, Kristina Hanspers (Gladstone Institutes, San Francisco) </p>

<p>Contact: <a href="mailto:anders.riutta@gladstone.ucsf.edu" >anders.riutta@gladstone.ucsf.edu</a></p>

<br />
<h2 dir="ltr" >Idea 17: Implementation of new clustering techniques into Cytoscape</h2>

<p>clusterMaker is a Cytoscape App that provides users with a number of clustering algorithms and visualizations. This project involves extending clusterMaker through adding new algorithms or visualizations. Some possible new attribute-based algorithms include biclustering, DBSCAN, model-based clustering, tight clustering. New network partitioning algorithms include PBD (Pujol, 2006), SCIPi (Jiang &amp; Singh 2010), Ensemble clusters, or any one of many other recently published algorithms. There is also lots of room for improvements to the visualization of clusters, including time-series morphing of heatmaps, generation of difference or merged heatmaps, or tools to more easily navigate clusters after computation.</p>

<br />
<p>Language and Skills: Java, and a strong math background</p>

<p>Potential Mentors: Scooter Morris (UCSF)</p>

<p>Contact: <a href="mailto:scooter@cgl.ucsf.edu" >scooter@cgl.ucsf.edu</a> </p>

<br />
<h2 dir="ltr" >Idea 18: Drag &amp; Drop or Palette Editor for SBGN and other graphical notations in Cytoscape</h2>

<p>The goal is to design and implement a drag-and-drop or palette-based editor for Cytoscape that would include nodes, edges, and graphical annotations. In the ideal case, this would be extensible so that apps add their own nodes or edges with specific graphical attributes. The Systems Biology Graphical Notation (SBGN) is a standard graphical notation for presenting pathways and protein interactions. Currently, Cytoscape lacks the node symbols and edge types required for SBGN. In this project, those notations will be added to the Cytoscape core, and a simple palette editor developed for user interaction.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Scooter Morri, Alex Pico</p>

<p>Contact: <a href="mailto:scooter@cgl.ucsf.edu" >scooter@cgl.ucsf.edu</a> </p>

<br />
<h2 dir="ltr" >Idea 19: Improve and expand Cytoscape’s color mapping options</h2>

<p>Currently, the color options for Cytoscape’s discrete color mappings are very limited. Brewer palettes provide a luminosity-neutral set of colors that would be extremely useful to add to Cytoscape, and there are several other approaches to choosing sets of colors that contrast nicely. Creating these color palettes and providing them for use within Cytoscape’s visual style interface would be extremely useful to Cytoscape users.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Scooter Morris, Michael Heuer, Alex Pico</p>

<p>Contact: <a href="mailto:scooter@cgl.ucsf.edu" >scooter@cgl.ucsf.edu</a> </p>

<br />
<h2 dir="ltr" >Idea 20: Network alignment app for Cytoscape</h2>

<p>One of the increasingly common needs when analyzing biological networks is network comparison. Often this involves aligning two or more networks so that the same nodes are in the same place and differences between networks can be highlighted. This could involve time-series data, host-pathogen interaction data, or cross-species comparisons. There are several algorithms that provide this functionality, but none have been implemented in Cytoscape. This project would involve implementing one or more network alignment algorithms and providing visualizations to allow users to explore the similarities and differences between the aligned networks.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Scooter Morris</p>

<p>Contact: <a href="mailto:scooter@cgl.ucsf.edu" >scooter@cgl.ucsf.edu</a> </p>

<br />
<h2 dir="ltr" >Idea 21: Constrained layout algorithms in Cytoscape</h2>

<p>Most layout algorithms in Cytoscape rely on inherent network topology and only a crude set of spatial constraints (e.g., grid, circle). There are many cases when working with networks in Cytoscape where you want to be able to constrain layouts to user- or attribute-defined spaces. This idea involves implementing layouts that are constrained by:</p>

<ul>
<li><p>specific shapes </p>

</li>
<ul>
<li><p>basic shapes </p>

</li>
<li><p>cellular templates (e.g., with subcellular organelles)</p>

</li>
<li><p>attribute-driven Venn diagrams (e.g., to visualize attribute groupings and overlaps)</p>

</li>
</ul>

<li><p>specific “anchor” nodes</p>

</li>
<ul>
<li><p>user-defined (e.g., manually selected)</p>

</li>
<li><p>attribute-defined (e.g., hubs or column values)</p>

</li>
</ul>

<li><p>etc...</p>

</li>
</ul>

<br />
<p>Language and Skills: Java, and a strong math background</p>

<p>Potential Mentors: Scooter Morris, Alex Pico, Bruce Aronow</p>

<p>Contact: <a href="mailto:scooter@cgl.ucsf.edu" >scooter@cgl.ucsf.edu</a> </p>

<br />
<h2 dir="ltr" >Idea 22: A SPARQL query plugin for Pathvisio</h2>

<p>Background: The semantic web is becoming a valuable resource for Pathway knowledge. The challenge still remain to smoothly integrate this knowledge into biological frameworks. With their SPARQL endpoints Uniprot (<a href="http://beta.sparql.uniprot.org/" >http://beta.sparql.uniprot.org/</a>) or the EBI rdf Platform (<a href="https://www.ebi.ac.uk/rdf/" >https://www.ebi.ac.uk/rdf/</a>) , biological data is becoming available as semantic web data. To use this knowledge one need not only to learn SPARQL queries, but also be prepared to convert the semantic data in different formats that comes with the various biological platforms. Using semantic web data in Pathvisio, we currently need to compose the appropriate SPARQL queries, get the data and write various parser to change the data in to the different data models. </p>

<br />
<p>Goal: To make a Pathvisio plugin that allows a pathway curator to write SPARQL queries against a variety of semantic web based pathways resources and include the obtained triples into the Pathway under scrutiny. The plugin should also provide an interface to SPARQL results for other Pathvisio plugins. Having such SPARQL interface still requires a pathway editor to become knowledgeable in writing SPARQL queries. The burden of having to write various parsers for semantic web data however, is taken away. The plugin should:</p>

<ul>
<li><p>Provide a SPARQL query editor, from where the user could compose queries, which are then subsequently launched to the respective SPARQL endpoints.</p>

</li>
<li><p>Provide an interface that wrap specific SPARQL queries for other Pathvisio plugins. </p>

</li>
<li><p>Link to a centralized repository (e.g. Github) where SPARQL queries can be stored for future reference and others to use.</p>

</li>
<li><p>Link to a centralized index of relevant SPARQL endpoints. This index might again be stored on platforms like Github. </p>

</li>
<li><p>Import the resulting triples, which can then be linked to recognizable pathway components, or as data elements to use in Pathway analysis. </p>

</li>
<li><p>Make a RDF representation of a Pathway opened in Pathvisio. These triples are to be used as template for SPARQL queries. The user should be able to click on a triple which is then added to the WHERE block in the SPARQL query.</p>

</li>
</ul>

<br />
<p>Potential students: This idea is for a student who has solid background in SPARQL queries, or the Semantic web in general, and who is looking into opening up the Semantic Web to specific knowledge domains (I.e. Biology and/or Medicine). </p>

<br />
<p>Language and Skills: SPARQL, RDF, and Java</p>

<p>Potential Mentors: Andra Waagmeester, Anders Riutta, Martina Kutmon </p>

<p>Contact: <a href="mailto:wikipathways-discuss@googlegroups.com" >wikipathways-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 23: Graph Database Support for Cytoscape 3 by TinkerPop Software Stack</h2>

<p><a href="http://www.cytoscape.org/cy3.html" >Cytoscape 3</a> is now available to end-users, and we need an easy-to-use Cytoscape App which provides the following functions to support huge data sets:</p>

<ul>
<li><p>Mount huge network(s) in disk-based graph database</p>

</li>
<ul>
<li><p><a href="http://www.neo4j.org/" >Neo4j</a></p>

</li>
<li><p><a href="http://www.openrdf.org/" >Sail</a></p>

</li>
</ul>

<li><p>Send query to the network</p>

</li>
<ul>
<li><p>Search by simple properties (Name, ID, etc.)</p>

</li>
<li><p><a href="http://en.wikipedia.org/wiki/SPARQL" >SPARQL</a> (for semantic data)</p>

</li>
<li><p><a href="https://github.com/tinkerpop/gremlin/wiki" >Gremlin</a></p>

</li>
</ul>

<li><p>Visualize the result</p>

</li>
</ul>

<p>To support multiple databases, you need to use<a href="https://github.com/tinkerpop/blueprints" > TinkerPop Blueprints</a>. The goal of this project is publish a maintainable and usable (by average biologists) App for all Cytoscape 3 users.</p>

<br />
<p>Language and Skills: Java, Maven, OSGi, Git/GitHub</p>

<p>Potential Mentors: Kei Ono, Samad Lotia</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 24: Virtual Cell iPhone app enhancements</h2>

<p>The Virtual Cell modeling and simulation framework is an open source software for modeling and simulations of biological systems providing both client- and server- based modeling and simulation capabilities. An iOS app has been prototyped which interfaces with the Restful VCell API and allows mobile access to VCell models and management of running simulations within our a batch environment on a centralized computational facility. This app must browse private and public models, monitor and control simulation jobs, manage user quotas, and provide simple results viewing. During the project period, we will reevaluate the requirements of the production software, refine the design, add features and beta test with our team and VCell users.</p>

<br />
<p>Language and Skills: Objective-C/iOS</p>

<p>Possible mentors: Jim Schaff</p>

<p>Contact: <a href="mailto:vcell-discuss@googlegroups.com" >vcell-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 25: Web publishing of models from Virtual Cell database</h2>

<p>The<a href="http://vcell.org" > Virtual Cell</a> modeling and simulation framework is an open source software for modeling and simulations of biological systems using Java RMI, providing both client- and server- based modeling and simulation capabilities. The biologically oriented user interface allows experimentalists to create models, define cellular geometry, specify simulations and analyze the simulation results. The models and results are stored on a remote server in ORACLE database and presently can be reviewed only in the VCell software. You need to implement a content management system that will search and retrieve model information from ORACLE or MySQL database and publish it online. The model structure stored in a relationship database is available at<a href="http://vcell.org/technology/VcmlSpec.html" > http://vcell.org/technology/VcmlSpec.html</a></p>

<br />
<p>Language and Skills: Java</p>

<p>Possible Mentors: Michael Blinov, Jim Schaff</p>

<p>Contact: <a href="mailto:vcell-discuss@googlegroups.com" >vcell-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 26: Porting Interference App to Cytoscape 3 with new features for undirected and directed networks</h2>

<p>Background: Interference is a Cytoscape plug-in for virtual knock-out experiments on complex networks. It allows to evaluate the topological effects of single or multiple nodes removal from a network. In this perspective, interference allows virtual node knock-out experiments: it is possible to remove one or more nodes from a network and analyze the consequences on network structure, by looking to the variations of the node centralities values. As the centrality value of a node is strictly dependent on the network structure and on the properties of other nodes in the network, the consequences of a node deletion are well captured by the variation on the centrality values of all the other nodes. With the interference app the user can:</p>

<ul>
<li><p>Simulate pharmacological treatment: one can potentially predict side effects of the drug by looking at the topological properties of nodes in a drug-treated network, meaning with that a network in which a drug-targeted node (protein) was removed. To inhibit a protein (for instance a kinases) corresponds to removing the node from the network</p>

</li>
<li><p>Simulate gene deletion: gene deletion implies losing encoded proteins, thus resulting in the corresponding removal of one or more nodes from a protein network</p>

</li>
</ul>

<br />
<p>Goal: The Interference plugin is based on some features of the CentiScaPe plugin, and is available for Cytoscape 2.8.x but not yet for Cytoscape 3. The Idea is to port the plugin to Cytoscape 3 and to add some features. The plugin works for undirected networks, but now a new version of the CentiScaPe plugin allows to compute several centralities also for directed networks. The new tool should:</p>

<ul>
<li><p>Work with Cytoscape 3.</p>

</li>
<li><p>Provide the Eigenvector, Bridging Centrality and Edge Betweenness</p>

</li>
<li><p>centrality Interference (now missing).</p>

</li>
<li><p>Provide the directed Interference for all the centralities implemented</p>

</li>
<li><p>More info at:<a href="http://www.cbmc.it/%7Escardonig/interference/Interference.php" > http://www.cbmc.it/%7Escardonig/interference/Interference.php</a></p>

</li>
</ul>

<br />
<a href="http://www.cbmc.it/%7Escardonig/interference/Interference.php" ></a><p>Language and Skills: Java</p>

<p>Potential Mentors: Giovanni Scardoni (CBMC, University of Verona),Fausto Spoto (University of Verona)</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 27: Porting Pesca App to Cytoscape 3 with new features for undirected and directed networks</h2>

<p>Background: The Pesca plugin is able to generate networks extracting shortest-path based subnetwork in order to help reconstructing biological networks. Particularly it provides the shortest path tree from a node, the shortest path cluster and a function to connect single nodes to giant component based on the shortest path from the nodes to the giant components.</p>

<br />
<p>Goal: These features are available for Cytoscape 2.8.x in beta version. The Idea is to port all the features to Cytoscape 3 and to add new features to the app. Particularly:</p>

<ul>
<li><p>Provide the shortest path tree and shortest path cluster for both directed and undirected networks</p>

</li>
<li><p>Provide the connection from single nodes to a giant component for both directed and undirected networks</p>

</li>
<li><p>Provide the possibility to download directed and undirected human interactome to be used to generate biological networks (from the human ppi network database:<a href="http://dp.univr.it/%7Elaudanna/LCTST/downloads/index.html" > http://dp.univr.it/~laudanna/LCTST/downloads/index.html</a>)</p>

</li>
<li><p>Provide a user friendly output allowing to easily extract and generate subnetworks</p>

</li>
<li><p>Provide some simple statistics (S-P distribution, number of occurrence of nodes)</p>

</li>
</ul>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Giovanni Scardoni (CBMC, University of Verona), Fausto Spoto (University of Verona)</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 28: New Features for Online Tutorial System</h2>

<p><a href="http://http//opentutorials.cgl.ucsf.edu" >OpenTutorials</a> is a free, online tutorial resource for open source and NRNB-supported tools. It is built on top of MediaWiki and a handful of extensions. It’s basic function is to provide a wiki-based collection of tutorials that can be viewed as a web page, slide show or printed handout. The wiki approach makes it easy to build and maintain content. The slide show and handout features make it useful for presenters.</p>

<p>There are a number of features that need to be implemented to improve the site and really make it effective to a larger community. What we’re looking for here with this project is an application that is very clear about the use cases of tutorials:</p>

<ul>
<li><p>for developers wanting to create and maintain tutorial materials</p>

</li>
<li><p>for presenters of tutorials</p>

</li>
<li><p>and for users learning from tutorials.</p>

</li>
</ul>

<br />
<p>The application should include a number of features that enhance these use cases. When possible, the features should point to other examples were similar approaches have proven successful (e.g., in other popular software). After we discuss the features, your application should then include implementation details and a timeline. These would be implemented as one or more PHP extensions, for example. Experience or some demonstration of familiarity with MediaWiki or PHP would be a plus. Here are some feature ideas to get you started:</p>

<ul>
<li><p>Mix-and-match: We need a mechanism to allow presenters to mix and match tutorials (or sections of tutorials) into sets that are tailored for their class or workshop audience.</p>

</li>
<li><p>Transient locks: We need a mechanism for temporarily locking content while it is being presented (so that someone isn’t editing a wiki page while you are presenting it to a class of 100 students!). This can be done either by supporting temporary presenter “copies” that are not publicly editable, or simply locks that presenters can set for specific dates and times.</p>

</li>
<li><p>Enhanced image replacement tools to make it easier to update screenshots.</p>

</li>
<li><p>Tutorial builder tools using templates and forms to get a new tutorial started, or an integrated html-to-wiki converter like<a href="http://labs.seapine.com/htmltowiki.cgi" > this</a>.</p>

</li>
<li><p>Tweaks to the wiki page display of header/meta data and of slide breaks (e.g., a clear, yet unobtrusive icon).</p>

</li>
</ul>

<br />
<p>Language and Skills: PHP, MediaWiki</p>

<p>Potential Mentors: Alex Pico, Kristina Hanspers</p>

<p>Contact: <a href="mailto:apico@gladstone.ucsf.edu" >apico@gladstone.ucsf.edu</a></p>

<br />
<h2 dir="ltr" >Idea 29: Custom Export Editor for WikiPathways Using SVG</h2>

<p>Background:<a href="http://wikipathways.org" > WikiPathways</a> is a collaborative pathway editing and sharing platform. There are many ways to use the pathway content including data visualization and analysis. Another use is as illustrations in presentations and publications. We currently provide SVG export for this purpose. However, the images usually require quite a bit of “touch up” in secondary tools like Illustrator, which inhibits this use case.</p>

<p>Goal: Customize a basic SVG Editor to insert into the export process to allow basic visual style updates and custom annotations on top of the pathway image. Pre-configured options would include:</p>

<ul>
<li><p>thicker lines and bold font (for better display in powerpoint)</p>

</li>
<li><p>swap basic cellular component shapes with nice graphics (provided from an existing repository)</p>

</li>
<li><p>color schemes</p>

</li>
</ul>

<p>This simple “touch up” editor could dramatically increase the audience for WikiPathways to include anyone looking for an easy tool to generate nice looking pathway images for presentations. Ideally, this editor would work on tablets as well as across major browsers.</p>

<br />
<p>Language and Skills: SVG,<a href="http://code.google.com/p/svg-edit/" > SVG-edit</a> (or alternative)</p>

<p>Potential Mentors: Alexander Pico, Anders Riutta</p>

<p>Contact: <a href="mailto:wikipathways-discuss@googlegroups.com" >wikipathways-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 30: Integration of WikiPathways Visualization with PathVisio into Bioclipse</h2>

<p>Goal: To integrate PathVisio visualization widget into Bioclipse for visualization of pathway information from WikiPathways. Selecting protein and metabolites in the pathway visualization, will shows of that chemical entity in other Bioclipse components, using the Jmol and JChemPaint plugins for Bioclipse. Preferably, interaction can be scripted, allowing integration with other metabolomics features in Bioclipse.</p>

<br />
<p>Motivation: Bioclipse is a powerful platform combining a powerful scripting language to write data analysis and visualization workflows, which can easily be shared on, for example, MyExperiment.org [1]. While it has various components to support metabolomics in general, it lacks support for visualization of metabolic pathways.</p>

<br />
<p>Description: You will use Java to write a plugin for Bioclipse [2] that uses PathVisio [3] as a library to visualize metabolic pathways from WikiPathways inside Bioclipse. You will use the Eclipse selection event propagation mechanism [4] so that when users select protein and metabolites, Bioclipse will automatically visualize 2D or 3D structural information about those molecules (only the event creation has to be implemented). Preferably, a scripting language extension is written [5], to allow extracting information from the pathway and highlighting of parts of the pathway. This involves extending an existing interface with methods that can be run from the scripting language.</p>

<ol ><li><p><a href="http://myexperiment.org/" >myexperiment.org</a></p>

</li>
<li><p><a href="http://www.bioclipse.net/" >www.bioclipse.net</a></p>

</li>
<li><p><a href="http://www.pathvisio.org/" >www.pathvisio.org</a></p>

</li>
<li><p><a href="http://www.eclipse.org/articles/Article-WorkbenchSelections/article.html" >www.eclipse.org/articles/Article-WorkbenchSelections/article.html</a></p>

</li>
<li><p><a href="http://www.biomedcentral.com/1471-2105/10/397" >www.biomedcentral.com/1471-2105/10/397</a></p>

</li>
</ol><br />
<a href="http://www.biomedcentral.com/1471-2105/10/397" ></a><p>Language and Skills: Java</p>

<p>Potential Mentors: Egon Willighagen, Jonathan Alvarsson</p>

<p>Contact: <a href="mailto:wikipathways-discuss@googlegroups.com" >wikipathways-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 31: Integration of Cytoscape into Bioclipse</h2>

<p>Goal: To integrate the new Cytoscape OSGi visualization widgets into Bioclipse for visualization of large interaction networks. Selecting protein and metabolites in the pathway visualization, will shows of that chemical entity in other Bioclipse components, using the Jmol and JChemPaint plugins for Bioclipse. Preferably, interaction can be scripted, allowing integration with other metabolomics features in Bioclipse.</p>

<br />
<p>Motivation: Bioclipse is a powerful platform combining a powerful scripting language to write data analysis and visualization workflows, which can easily be shared on, for example, MyExperiment.org [1]. While it has been used in drug discovery and predictive toxicology in pharmaceutical industry [2], it currently does not have functionality to visualize large graph structures as Cytoscape has.</p>

<br />
<p>Description: You will use Java to write a plugin for Bioclipse [3] that uses Cytoscape. No particular graph is aimed at, but the plugin will use the Eclipse selection event propagation mechanism [4] so that when users select protein and metabolites, Bioclipse will automatically visualize 2D or 3D structural information about those molecules (only the event creation has to be implemented). Preferably, a scripting language extension is written, to allow extracting information from the visualized graphs.</p>

<ol ><li><p><a href="http://myexperiment.org/" >myexperiment.org</a></p>

</li>
<li><p><a href="http://pubs.acs.org/doi/abs/10.1021/ci200242c" >pubs.acs.org/doi/abs/10.1021/ci200242c</a></p>

</li>
<li><p><a href="http://www.bioclipse.net/" >www.bioclipse.net</a></p>

</li>
<li><p><a href="http://www.eclipse.org/articles/Article-WorkbenchSelections/article.html" >www.eclipse.org/articles/Article-WorkbenchSelections/article.html</a></p>

</li>
</ol><br />
<a href="http://www.eclipse.org/articles/Article-WorkbenchSelections/article.html" ></a><p>Language and Skills: Java</p>

<p>Potential Mentors: Egon Willighagen, someone from Bioclipse team</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 32: PathVisio Plugin to Load Additional Information about Data Nodes</h2>

<p>Ensembl now provides a BioMart implementation for Ensembl genes, variation and regulation. The goal of this project is to develop a plugin that loads additional information about the data nodes in a pathway and displays them in a side panel (gene name, chromosome location, strand, %GC content, transcript count, etc). This could be extended with other services that provide other information about the data nodes.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Martina</p>

<p>Contact: <a href="mailto:wikipathways-discuss@googlegroups.com" >wikipathways-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 33: Publish Cytoscape 3 Session to cytoscape.js</h2>

<p><a href="http://www.cytoscape.org/cy3.html" >Cytoscape 3</a> and<a href="http://cytoscape.github.com/cytoscape.js/" > cytoscape.js</a> are completely independent software packages; 3 is a Java Desktop application and .js is a JavaScript library to visualize network data. However, it shares some basic concepts, such as Visual Styles. It is very useful if we can reproduce same/similar Cytoscape 3 network views in web browsers to share/publish data sets.</p>

<p>The goal of this project is exporting Cytoscape 3 data as JSON, and reproducing same/similar interactive network views with cytoscape.js and other JavaScript libraries.</p>

<ul>
<li><p>Export Cytoscape 3 data as JSON</p>

</li>
<ul>
<li><p>Networks</p>

</li>
<li><p>Data Tables</p>

</li>
<li><p>Visual Styles</p>

</li>
</ul>

<li><p>This is a JAVA programming project. JavaScript knowledge is optional, but Java/OSGi knowledge is REQUIRED. You need to write a lot of Java classes to convert Cytoscape 3 data objects into JSON using Jackson library.</p>

</li>
<li><p><a href="https://github.com/idekerlab/json-impl" >Prototype is available</a></p>

</li>
<li><p>Create a test HTML5 page to visualize Cytoscape 3 network views with cytoscape.js</p>

</li>
<li><p>(Optional) Add import JSON function for Cytoscape 3</p>

</li>
</ul>

<br />
<p>Language and Skills: Java, JavaScript, HTML5, Git/GitHub</p>

<p>Potential Mentors: Kei Ono, Christian Lopes, Max Franz, Gary Bader</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 34: BridgeDb config plugin in PathVisio</h2>

<p><a href="http://www.bridgedb.org" >BridgeDb</a> is an identifier mapping framework used in PathVisio and WikiPathways. Currently it is possible to load a mapping databases for gene products and metabolites. The existing<a href="http://www.pathvisio.org/wiki/BridgeDbConfigPluginHelp" > BridgeDb config plugin</a> provides functionality to load more mapping databases or to use other webservices for the mappings. This functionality should be extended and integrated into the core PathVisio application as another core module. The configuration dialog has to be improved so it is more intuitive and user friendly.</p>

<p>The main features of this module should be:</p>

<ul>
<li><p>user friendly dialog to configure mapping services</p>

</li>
<li><p>centralized module for identifier mapping in PathVisio</p>

</li>
<li><p>system to define which databases are used for which species and/or data node type</p>

</li>
</ul>

<br />
<p>Language and Skills: Java, OSGi, BridgeDb</p>

<p>Potential Mentors: Martina Kutmon, Anwesha Bohler</p>

<p>Contact: <a href="mailto:wikipathways-discuss@googlegroups.com" >wikipathways-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 35: Publication Reference Plugin for PathVisio</h2>

<p>Currently<a href="http://www.pathvisio.org" > PathVisio</a> supports publication reference annotation of pathways, data nodes and interactions with PubMed identifiers. The goal of this plugin is to provide a more advanced framework for reference annotations.</p>

<p>This plugin should provide (can be extended):</p>

<ul>
<li><p>functionality to use other identifiers than pubmed (e.g. DOI) (might be possible to use an existing java library for that (e.g. Jabref))</p>

</li>
<li><p>show all references in a side tab with linkouts</p>

</li>
<li><p>exporting all references in different reference style formats</p>

</li>
</ul>

<br />
<p>Language and Skills: Java, OSGi</p>

<p>Potential Mentors: Sravanthi Sinha, Martina Kutmon</p>

<p>Contact: <a href="mailto:wikipathways-discuss@googlegroups.com" >wikipathways-discuss@googlegroups.com</a></p>

<br />

<h2 dir="ltr" >Idea 36: Improved Layering/Interaction in Cytoscape 3 Networks</h2>

<p>This project involves working on the visualization core in Cytoscape 3 (ding) to significantly improve the user experience. Currently, Cytoscape supports a single canvas to visualize the network. When moving or changing graphic objects (nodes and edges), the entire visible portion of the canvas needs to be updated. It has been pointed out by visualization experts that this problem can be easily solved by adding an additional “selection” canvas. Motion of nodes and edges could be performed in this canvas rather than the main canvas, significantly improving the interactivity on dense networks. The addition of a “selection” canvas also enables a number of visualization techniques that are too computationally expensive in the current Cytoscape model.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Scooter Morris</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 37: Compound Graph Visualization in Cytoscape 3</h2>

<p>Cytoscape 3 has introduce two concepts into the core, groups and annotations, that could be combined to provide a compound graph visualization. Currently groups can be either in a “collapsed” or “expanded” state. For a compound graph, a new visualization state needs to be introduced that allows the compound node (group) to be visualized as a shape (concave or convex hull) around it’s children. Moving that shape would move all of it’s children, etc.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Scooter Morris</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 38: Network Analysis Enhancements</h2>

<p>Cytoscape 3 includes a Network Analyzer bundle that performs a number of extremely useful network analyses. Unfortunately, the functionality provided is not exposed in a way that other plugins can utilize. This project will enhance the network analysis functionality to support calculating individual network statistics using an exported API or exported Cytoscape tasks. Moreover, the project will provide this functionality in both a GUI and non-GUI (headless) mode.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Scooter Morris and Nadezhda Doncheva</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 39: Visualization of Time-series Cluster Data</h2>

<p>Currently, the clusterMaker plugin uses standard heat maps to visualize attribute cluster data (based on Java TreeView). Increasingly, time series data is becoming important for clustering applications and it would be useful to expand the provided visualization options to explicitly support time series data. For these purposes, we’re thinking about time-series that have multiple conditions in each series, which means we need to move beyond the standard two-dimensional heat map. This project can explore utilizing animation, small multiples, or other standard visualization techniques (or your own novel techniques) to provide new visualization opportunities to users. Ideally, this would be integrated with network-based animation tools such as DynNetwork or CyAnimator.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Scooter Morris</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 40: Network Animation Tools</h2>

<p>CyAnimator was a key-frame style animation tool for Cytoscape 2.8. This project will implement a similar tool for Cytoscape 3, taking advantage of the existing CyAnimator where appropriate. This tool is distinct from the DynNetwork tool in that it is not necessarily driven by specific data, although one possible use case would be to create dynamic XGMML as an output from the tool.</p>

<br />
<p>Language and Skills: Java</p>

<p>Potential Mentors: Scooter Morris</p>

<p>Contact: <a href="mailto:cytoscape-discuss@googlegroups.com" >cytoscape-discuss@googlegroups.com</a></p>

<br />
<h2 dir="ltr" >Idea 41: Touch-based Exploration of Large Data Collections and Networks</h2>

<p>The goal of this project is to develop a general, open source framework for performing iterative, faceted search on large data collections and networks specifically for touch-based input devices. The project will leverage ideas from existing desktop applications (see<a href="http://www.youtube.com/watch?v=BZuFUZpEZ-A" > Microsoft Pivot</a>) but will include experimentation and innovation in interaction techniques in order to create a more engaging user experience. Ultimately the framework will be applied in one of potentially many biological applications, including MedSavant, which is described below.</p>

<p>MedSavant is a high-performance search engine for genomic datasets, which is primarily aimed to help researchers identify genetic conditions that cause disease. The desktop client allows users to</p>

<p>incrementally construct faceted search queries and visualize their results virtually in realtime. Similar workflows enabled by the desktop client will be made possible by integration with the proposed</p>

<p>framework. For more information, visit <a href="http://www.medsavant.com" >www.medsavant.com</a>.</p>

<br />
<p>Language and Skills: Android or iOS development; AJAX, JSON, Java, and Javascript; UNIX</p>

<p>Potential Mentors: Marc Fiume, Marta Girdea</p>

<p>Contact: <a href="mailto:mfiume@cs.toronto.edu" >mfiume@cs.toronto.edu</a></p>

<br />
<h2 dir="ltr" >Idea 42: Improve the Text-Mining Functionality of Factoid</h2>

<p><a href="http://pathwaycommons.github.io/factoid/" >Factoid</a> aims to be a tool that helps authors annotate biological interactions that occur in the text of academic publications in a machine-readable manner. Currently, Factoid uses basic entity-recognition to identify genes, proteins, biological processes to identify interactions between these entities. The goal of this project would be to integrate more sophisticated algorithms into the extraction of interactions from text content. A starting point for some of these would be the algorithms outlined by the EBI website WhatIzIt.</p>
<br />
<p>Languages and Skills: Experience connecting web services, Javascript</p>

<p>Potential Mentors: Augustin Luna (MSKCC), Max Franz (U of Toronto) </p>

<p>Contact: <a href="mailto:lunaa@cbio.mskcc.org" >Augustin Luna</a>, <a href="mailto:maxkfranz@gmail.com" >Max Franz</a></p>

<!-- END PASTE -->

    </div> <!-- .span9 !-->
</div> <!-- .row !-->

<script src="../assets/js/jquery-1.11.0.min.js"></script>
<script src="../assets/js/jquery-ui.min.js"></script>
<script src="../assets/js/jquery.tocify.min.js"></script>
<script>
$("#nav").tocify({
    context: "#ideas",
    showAndHide: false,
    selectors: "h2",
});
</script>
</body>
</html>
